bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -631,12 +631,13 @@ class PlotCanvasView extends Renderer.View
   _map_hook: (ctx, frame_box) ->
 
   _paint_empty: (ctx, frame_box) ->
-    @visuals.border_fill.set_value(ctx)
-    ctx.fillRect(0, 0,  @canvas_view.mget('width'), @canvas_view.mget('height'))
-    ctx.clearRect(frame_box...)
-    @visuals.background_fill.set_value(ctx)
-    ctx.fillRect(frame_box...)
-
+    ctx.clearRect(0, 0,  @canvas_view.mget('width'), @canvas_view.mget('height'))
+    if @visuals.border_fill.doit
+      @visuals.border_fill.set_value(ctx)
+      ctx.fillRect(0, 0,  @canvas_view.mget('width'), @canvas_view.mget('height'))
+    if @visuals.background_fill.doit
```

Comment:
I probably picked it up from here: http://www.mathworks.com/help/symbolic/mupad_ref/bordercolor.html

If there are concrete proposals for comprehensive, consistent naming schemes that we can adopt that are better, let's make them, and soon (of course `border_fill_color`, etc. would have to get a long deprecation)